src/main/scala/io/iohk/ethereum/blockchain/sync/RegularSync.scala

```diff
@@ -53,6 +54,19 @@ trait RegularSync {
       //FIXME: Decide block propagation algorithm (for now we send block to every peer) [EC-87]
       peersToDownloadFrom.keys.foreach(_.ref ! block)
 
+    //todo improve mined block handling - add info that block was not included because of syncing
+    //we allow inclusion of mined block only if we are not syncing / reorganising chain
```

Comment:
is there a separate task created for it (just so that we remember to fix this later)?